bochs/iodev/usb/scsi_device.cc

```diff
@@ -789,6 +797,166 @@ Bit32s scsi_device_t::scsi_send_command(Bit32u tag, Bit8u *buf, int lun, bool as
       BX_INFO(("Prevent Allow Medium Removal (prevent = %d)", buf[4] & 3));
       locked = buf[4] & 1;
       break;
+    case 0x4a: // SCSI_CD_EVENT_STATUS
+      {
+        Bit8u Class = 0;
+        Bit16u out_len = 4; // the initial header is 4 bytes
+        Bit8u *p = &outbuf[4];  // starts at byte 4
+        
+        // This code is experimental. may need some help
+        BX_DEBUG(("Event Status notification: Requested Byte 0x%02X", buf[4]));
+        
+        len = (Bit32s) (((Bit32u) buf[7] << 8) | buf[8]);
+        if (buf[1] & 1) { // polled bit is set
+          // only do the Event Header if length <= 4, else we can do the Event
+          // buf[4] can be zero. Usually the Guest is requesting the support class event bitmap when this happens.
+          if (len > 4) {
+            // hightest priority (lowest bit set) gets executed (only)
+            if (buf[4] & (1<<0)) { // Reserved bit is set (MMC4R05A.PDF, 6.7.1.3, page 242, not considered an error)
+              BX_DEBUG(("Event Status Notify: bit 0 in the request byte is set"));
+              Class = 0;
+            }
+            if (buf[4] & (1<<1)) { // Opertaional Change request bit is set
```

Comment:
typo in comment
also I suggest to have nice enums instead of using magic const bitmasks like (1<<2)
these bits have names - you even mention them in comments

Reply:
Indeed. You are right. My apologizes. Would you like a new pull request now, or may I wait until my next group of updates?

Reply:
Please keep updating the same PR. This is the easiest way to handle things.